challenges/classes.js

```diff
@@ -1,7 +1,54 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker1 {
+    constructor(attributes){
+    this.newLength = attributes.length;
+    this.newWidth = attributes.width;
+    this.newHeight = attributes.height;
+  }
+
+  volume = function() {
+    return this.newLength * this.newWidth * this.newHeight;
+  }
+
+  surfaceArea = function() {
+    return 2 * (this.newLength * this.newWidth + this.newLength * this.newHeight + this.newWidth * this.newHeight);
```

Comment:
Remember when you declare a method inside of a class you don't need to assign it to a variable. It should be something like `method() { }`